tests/textcase/telescope/options/enabled_methods_spec.lua

```diff
@@ -0,0 +1,38 @@
+local feature_flags = require("feature_flags")
+if not feature_flags.is_feature_available("telescope") then
+  return
+end
+
+local textcase = require("textcase")
+local test_helpers = require("tests.test_helpers")
+
+describe("Telescope options enabled_methods=...", function()
+  before_each(function()
+    textcase.setup({
+      enabled_methods = { "to_snake_case" },
+    })
+    require("telescope").load_extension("textcase")
+
+    local buf = vim.api.nvim_create_buf(false, true)
+    vim.api.nvim_command("buffer " .. buf)
+    vim.api.nvim_buf_set_lines(0, 0, -1, true, { "LoremIpsum" })
+  end)
+
+  it("should not register other keymappings", function()
```

Comment:
Did you mean `keymapping` in the context of Telescope enabled methods?

Reply:
Ah copy&paste mistake from the other enabled_methods spec 🙈

Reply:
I updated it.